### DISCUSSION TITLE
Online resources to learn about Linux, Windows and NASM

### DISCUSSION THREAD

**Discussion by ghost** [community/community#86613] [Programming Help] [answered]
### Body

What are some online resources I can use to learn about the OS and hardware so I can get better at Assembly. Also, can you recommend a website that someone with a lower IQ can use even though there probably isn’t one.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DTS-11** [comment, marked as answer]
Unfortunately, there are not so many good resources and books to learn nasm. But some of them I could recommend:

**Resources:**

- [official NASM documentation](https://www.nasm.us/doc/)
- [Low-Level Programming: C, Assembly, and Program Execution on Intel ® 64 Architecture](https://rads.stackoverflow.com/amzn/click/com/1484224027)
- [Assembly Language for Intel-Based Computers(Kip R. Irvine)](https://rads.stackoverflow.com/amzn/click/com/0132383101)
- [Introduction to 64 Bit Intel Assembly Language Programming for Linux(Benjamin Ray Seyfarth)](https://rads.stackoverflow.com/amzn/click/com/1478119209)
